loopchain/channel/channel_service.py

```diff
@@ -758,25 +666,57 @@ def score_invoke(self, _block: Block) -> dict or None:
             }
             transactions.append(transaction)
 
-        request = {
+        request_origin = {
             'block': {
                 'blockHeight': _block.header.height,
                 'blockHash': _block.header.hash.hex(),
                 'prevBlockHash': _block.header.prev_hash.hex() if _block.header.prev_hash else '',
                 'timestamp': _block.header.timestamp
             },
-            'transactions': transactions
+            'transactions': transactions,
+            'prevBlockGenerator': prev_block.header.peer_id.hex_hx() if prev_block.header.peer_id else '',
+            'prevBlockValidators': [rep['id'] for rep in self.__peer_manager.get_reps()]
         }
-        request = convert_params(request, ParamType.invoke)
+
+        if conf.ENABLE_IISS:
+            request_origin['isBlockEditable'] = hex(is_block_editable)
+
+        request = convert_params(request_origin, ParamType.invoke)
         stub = StubCollection().icon_score_stubs[ChannelProperty().name]
-        response = stub.sync_task().invoke(request)
+        response: dict = cast(dict, stub.sync_task().invoke(request))
         response_to_json_query(response)
 
-        tx_receipts = response["txResults"]
+        tx_receipts_origin = response.get("txResults", None)
```

Comment:
This is description of `dict.get(key[, default])`:
..__If default is not given, it defaults to None__.. 
(https://docs.python.org/3/library/stdtypes.html#dict.get)

... which means:
```suggestion
        tx_receipts_origin = response.get("txResults")
```
is more acceptable.